hs/t/y/Fix_Shadowed.txt

```diff
@@ -0,0 +1,6 @@
+
+Verifying knowledge assertions
+Verifying for generic connector
+  Verifying when ALL participants are honest
+  Verifying when NO participants are honest
+Checked 1 theorems; No failures!
```

Comment:
Is this correct with [the new version of `scripts/gen-test-output.sh`](https://github.com/reach-sh/reach-lang/pull/1095/files#diff-bfd62d82e917b4bb32e86f0a3d873ce7ca29b3bde6485b7c01f33b3dd6f6c784)?

You may need to `git rebase` to get it. You can also just copy and paste the script and run it.

Don't forget to do `brew install gnu-sed`!

You can direct message me on Slack if you need additional assistance with this.